### DISCUSSION TITLE
Problems with apt installation of wazuh-agent beta version 4.9.0

### DISCUSSION THREAD

**Discussion by ivan-ikl** [wazuh/wazuh#25048] [Q&A] [answered]
Greetings,

I have several Debian machines and would like to deploy the 4.9.0 beta version of Wazuh agents with support for journald. For starters, I tried installing it using the Ansible. However, I noticed that apt there did not manage to find any of the beta versions.

After this did not succeed, I followed the official manual installation instructions, but apt still didn't manage to find the beta version among available wazuh-agent versions. Is it possible to install the beta versions using apt (e.g. by using another Wazuh repository), or is an alternative type of installation preferred there? Could it be a temporary problem with the apt repository? Any help would be highly appreciated.

Thank you in advance!

Best regards,
Ivan

**Comment by MiguelazoDS** [comment, marked as answer]
Hello @ivan-ikl, 

You can use these packages, you will need also manager 4.9.0 

https://packages-dev.wazuh.com/pre-release/apt/pool/main/w/wazuh-agent/wazuh-agent_4.9.0-1_amd64.deb
https://packages-dev.wazuh.com/pre-release/apt/pool/main/w/wazuh-manager/wazuh-manager_4.9.0-1_amd64.deb

In case you also want to install the UI stack 

https://packages-dev.wazuh.com/pre-release/apt/pool/main/w/wazuh-indexer/wazuh-indexer_4.9.0-1_amd64.deb
https://packages-dev.wazuh.com/pre-release/apt/pool/main/w/wazuh-dashboard/wazuh-dashboard_4.9.0-1_amd64.deb

Take into account that this version may have some errors until is fully released. 

Regards!
